project-reports/2024/2024-annual-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,84 @@
+# 2024 Annual Review Hyperledger AnonCreds
+
+---
+layout: default
+title: 2024 Annual Review Hyperledger AnonCreds
+parent: 2024
+grand_parent: Project Updates
+---
+
+## Project Health
+
+_Guidance: Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights._
+
+Hyperledger AnonCreds LFX Insights Page for [Calendar Year 2023](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=anoncreds&repository=all&dateFilters=2023-01-01%20to%202023-12-31&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true) (v. [2022](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=anoncreds&repository=all&dateFilters=2022-01-01%20to%202022-12-31&dateRange=2022-01-01%20to%202022-12-31&compare=PP&granularity=month&hideBots=true))  and for [Q1 2024 Oct-Dec.](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=anoncreds&repository=all&dateFilters=2023-10-01%20to%202023-12-31&dateRange=2023-10-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true) (v. [Q4 2023](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=anoncreds&repository=all&dateFilters=2023-07-01%20to%202023-09-30&dateRange=2023-07-01%20to%202023-09-30&compare=PP&granularity=month&hideBots=true)).
```

Comment:
From the insights page, it appears that most of the work is being done by @swcurran. What can be done to improve the activity by others in the community?

Reply:
@ryjones -- can you check what repos are included in the AnonCreds LFX insights reports?  While I've done a lot with the spec, I've not done any coding and there has been a lot of code written this past year! I've done some of the code merging (after reviews by other devs), and perhaps that impacts the numbers?

Reply:
In the “Contributor Leaderboard”, I see me with 10% of the activities, followed by others at 8%, 5%, 3% and so on for at least 15 people with more than 1%.

I see the drop down with the list of repos. The only one I think is missing is anoncreds-v2-rs, so it is not that.

Love to have more participants 😄